mmv1/third_party/tgc_next/pkg/caiasset/asset.go

```diff
@@ -11,13 +11,13 @@ type Asset struct {
 	// The name, in a peculiar format: `\\<api>.googleapis.com/<self_link>`
 	Name string `json:"name"`
 	// The type name in `google.<api>.<resourcename>` format.
-	Type          string           `json:"assetType"`
+	Type          string           `json:"asset_type"`
 	Resource      *AssetResource   `json:"resource,omitempty"`
-	IAMPolicy     *IAMPolicy       `json:"iamPolicy,omitempty"`
-	OrgPolicy     []*OrgPolicy     `json:"orgPolicy,omitempty"`
+	IAMPolicy     *IAMPolicy       `json:"iam_policy,omitempty"`
+	OrgPolicy     []*OrgPolicy     `json:"org_policy,omitempty"`
 	V2OrgPolicies []*V2OrgPolicies `json:"v2_org_policies,omitempty"`
 	Ancestors     []string         `json:"ancestors"`
-	TfplanAddress []string         `json:"tfplanAddress,omitempty"`
+	TfplanAddress []string         `json:"tfplan_address,omitempty"`
 }
```

Comment:
Use the same json names with the [Asset](https://pkg.go.dev/cloud.google.com/go/asset/apiv1/assetpb#Asset) in cai api client